sdk/servicebus/service-bus/package.json

```diff
@@ -62,7 +62,6 @@
     "dotenv": "^7.0.0",
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^4.2.0",
-    "eslint-detailed-reporter": "^0.8.0",
```

Comment:
This change looks like a mistake, the reporter is used below in the lint script

Reply:
I think the package.json got reorganized automatically - https://github.com/Azure/azure-sdk-for-js/pull/3005/files#diff-7f408ecb2b8fde41a45e41ccd5944da2R95

Reply:
It's definitely not anywhere in here: https://github.com/Azure/azure-sdk-for-js/blob/6fde6edea1205503806f9980636da74435e3b7ca/sdk/servicebus/service-bus/package.json

Reply:
Thanks Brandon! I added it back

Reply:
Would be nice to sort it alphabetically but it's not critical